static/css/character_card_manager.css

```
@@ -6744,3 +6744,339 @@ margin-top: 10px;
+
+.card-assist-checkbox {
+    padding-top: 4px;
+    cursor: pointer;
+}
+
+.card-assist-row-key {
+    font-weight: 600;
+    color: #1f2937;
+    font-size: 14px;
+    padding-top: 4px;
+    word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)